chapter1/challenges/program/lib/password_validator.rb

```diff
@@ -3,5 +3,14 @@
 # - include at least one of these characters: !, @, $, %, &
 
```

Comment:
Having this comment closer to the if statement where the requirement is checked would make the code easier to follow.